api/src/ipc.c

```diff
@@ -109,6 +109,9 @@ static int ipc_waitfor(int (*cond)(uint32_t have, uint32_t expect), uint32_t wai
 {
     uint32_t completed_tokens;
     bool condition_met;
+    if (!wait_tokens) {
+        return UVISOR_ERROR_INVALID_PARAMETERS;
+    }
```

Comment:
What makes returning an error better than returning success?

One could expect waiting for no tokens to always succeed, as all tokens you wanted to wait for are complete already.

Reply:
Makes sense. Fixed.

Reply:
oops. git fail